mitodl/mit-open
# Positioning for search utils input close button

### What are the relevant tickets?

- Relates to https://github.com/mitodl/hq/issues/5472
- "Search input in filter facet (Topic/Department) formatting issues" in https://github.com/mitodl/hq/issues/5395
- This course-search-utils upgrade also fixes the issue with the page scrolling to the top on facet checkbox selection

### Description (What does it do?)


Upgrades to latest course-search-utils with expend icon fixes and centers the close icon in the search utils input.

The close positioning issue is in the main branch as well, but this change depends on https://github.com/mitodl/course-search-utils/pull/131, which replaces the Material `<i />` icons with Remixicon SVGs. 

Currently in Prod:

<img width="332" alt="image" src="https://github.com/user-attachments/assets/68ab31c3-74c9-44f8-9cd9-2b34943a7116">

With fix:

<img width="336" alt="image" src="https://github.com/user-attachments/assets/c3077369-8d2f-4d27-9187-e54aea8b033a">

and hover:

<img width="335" alt="image" src="https://github.com/user-attachments/assets/552b82f0-2d0b-4621-a597-1b04f04c050a">


### How can this be tested?


Yarn install and run. Close icon should appear as above.

This issue with the facet search inputs visbile when closed should also be fixed:
Before:
<img width="358" alt="image" src="https://github.com/user-attachments/assets/60a41f45-5c95-4560-a89d-25c620091f75">
After:
<img width="375" alt="image" src="https://github.com/user-attachments/assets/cb4a0a44-9316-4899-a622-c9c499379b63">



### Additional Context



We need to update the `@mitodl/course-search-utils` version once it's publish (it's currently pointing to the PR branch)
